ipatests/pytest_plugins/integration/tasks.py

```diff
@@ -29,6 +29,7 @@
 import time
 
 import dns
+import dns.resolver
```

Comment:
This import isn't used. I'm not sure why pylint didn't flag it.

Reply:
Confused me too. Maybe because of line 1131 (res = dns.resolver.Resolver()), however will remove it anyway.